templates/2025/talks/proposing_a_talk.html

```diff
@@ -43,26 +39,14 @@ <h2 class="h2 g-color-black g-font-weight-600">Proposing a Talk </h2>
 
                     <div class="col-md-12"><br>
                         <div class="col-md-12"> 
-                          <h2 class="h4 g-color-black g-font-weight-600 mb-3">
-                            {% if user.is_authenticated %}
-                            Hello{% if user.is_authenticated %} {% if request.user.user_profile %}{{ request.user.user_profile.name }}{% else %}@{{ request.user.username }}{% endif %}{% else %}{% endif %}, we are glad you are interested in speaking at PyCon Africa!<br> 
```

Comment:
I am currently working on #7 , i assuming that will unblock the sections on there that require/make use of authentication....